### PR TITLE
flutter: add test command

### PR DESCRIPTION
**Pull request by ev-john**
Page requested #4190

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/flutter.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Run Flutter on all started emulators and connected devices:
 
 `flutter run -d all`
+
+- Run tests in terminal from the root of the project:
```
Comment (navarroaxel) [inline pages/common/flutter.md:34]:
```suggestion
- Run tests in a terminal from the root of the project:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @ev-john welcome. Just a little fix.

**Comment by ev-john** [comment]
Thanks @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @ev-john :heart:

**Pull request merged**